manager/integration/tests/common.py

```diff
@@ -490,7 +493,7 @@ def delete_backup_volume(client, volume_name):
     wait_for_backup_volume_delete(client, volume_name)
 
 
-def create_and_check_volume(client, volume_name,
+def create_and_check_volume(client, volume_name, access_mode=ACCESS_MODE_RWO,
                             num_of_replicas=3, size=SIZE, backing_image="",
```

Comment:
I'm confused.  Wouldn't this require an added parameter in all those places that call with positional arguments (or maybe a change to use named arguments)?  Such as this one:  https://github.com/longhorn/longhorn-tests/blob/078c17d8ce66d649ce0d5ed68307cfa28549ef17/manager/integration/tests/test_basic.py#L1000

Reply:
Good catch, you are correct, this indeed messes up the arguments when optional arguments are invoked as positional arguments.